client/my-sites/plugins-wpcom/info-header.jsx

```diff
@@ -5,7 +5,7 @@ import Card from 'components/card';
 export const InfoHeader = React.createClass( {
 	render() {
 		return (
-			<Card>
+			<Card className="wpcom-plugins-info-header">
```

Comment:
Thanks for the tip. I had originally thought about that and then changed my mind to try and keep the components independent. Will update to the "dependency" though.